src/app/core/session/session-service/synced-session.service.ts

```diff
@@ -38,6 +41,7 @@ import { LoggingService } from "../../logging/logging.service";
  * [Session Handling, Authentication & Synchronisation]{@link /additional-documentation/concepts/session-and-authentication-system.html}
  */
 @Injectable()
+@UntilDestroy()
 export class SyncedSessionService extends SessionService {
```

Comment:
Some changes in this file can be undone if the `isInitial` function of the local session keeps returning a promise